modules/objc/examples/code_snippets/SampleCodeTest.m

```diff
@@ -1804,7 +1804,48 @@ - (void) dontTestQuerySyntaxJson {
 
 // end::query-syntax-pagination[]
 
+- (void) docsonly_QuerySyntaxN1QL() {
+    /* Documentation Only query-syntax-n1ql
+        Declared elsewhere: CBLDatabase* argDb
+    */
 
+    // tag::query-syntax-n1ql[]
+    NSError *error;
+
+    CBLDatabase *db == argDB;
+
+    NSString *n1qlstr = "SELECT * FROM _ WHERE type = \"hotel\""; // <.>
+
+    CBLQuery* thisQuery = [db createQuery: n1qlstr];
+
+    CBLQueryResultSet* resultset =  [thisQuery execute:&error];
+
+    // end::query-syntax-n1ql[]
+}
+
+- (void) docsonly_QuerySyntaxN1QLParams() {
+    /* Documentation Only query-syntax-n1ql-params
+        Declared elsewhere: CBLDatabase* argDB
+    */
+
+    // tag::query-syntax-n1ql-params[]
+    NSError *error;
+
+    CBLDatabase *db = argDB;
+
+    NSString *n1qlstr = "SELECT * FROM _ WHERE type = $type"; // <.>
+
+    CBLQuery* thisQuery = [db createQuery: n1qlstr];
+
+    CBLQueryParameters* n1qlparams = [[CBLQueryParameters alloc] init];
+    [params setString: @"hotel" forName: @"type"]; // <.>
+
+    thisQuery.parameters = n1qlparams;
```

Comment:
don't think this is correct! 


But something like the below is a way to set a dynamic variable value to a N1QL query. 
```
NSString* typeVar = @"hotel";
NSString *n1qlstr = [NSString stringWithFormat: @"SELECT * FROM dbName WHERE type = %@", typeVar];
```

Reply:
Hm. Neat trick. Not sure Jay. I thought that we were supposed to use the Query parameters ... maybe both work, but which do we want people using? I'll follow this up and change asap.

Reply:
I don't think this code will work, using QueryParams!